.github/workflows/rust_ci.yml

```diff
@@ -15,9 +14,7 @@ permissions:
 
 jobs:
   build:
-
-    runs-on: ubuntu-24.04
-
+    runs-on: ubuntu-latest
```

Comment:
For stability and auditability, I recommend we fix the version here (e.g. `ubuntu-24.04`)